src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,14 +1,15 @@
 package mate.academy.spring.dto.request;
 
+import lombok.Data;
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.Password;
+
+@Data
 public class UserRequestDto {
+    @Email
     private String email;
+    @Password
     private String password;
-
-    public String getEmail() {
-        return email;
-    }
-
-    public String getPassword() {
-        return password;
-    }
+    @Password
```

Comment:
this should be TYPE annotation, above the class, and check if the password and repeatPassword match